libpod/oci.go

```diff
@@ -348,7 +349,8 @@ func (r *OCIRuntime) createOCIContainer(ctr *Container, cgroupParent string) (er
 	childStartPipe.Close()
 
 	// Move conmon to specified cgroup
-	if r.cgroupManager == SystemdCgroupsManager {
+	if os.Getuid() != 0 {
```

Comment:
I really don't like the extra empty `if` here - if we want to do nothing if we're not root, we should wrap the `if r.cgroupManager` block in another `if`

Reply:
changed